### ISSUE TITLE
An information about uninstalling "explainshell-cli" package/software in ReadMe would have been nice.

### ISSUE THREAD

**Issue by PranavBhattarai** [p1g30n/explainshell-cli#2]
I wanted to test explainshell-cli and see how it works. I'm impressed.
Installation so easy & straightforward mechanism:

> cargo install --git https://github.com/p1g30n/explainshell-cli

But unfortunately, we don't have such an option to go back to the previous state, do we? Like:

> cargo uninstall --git https://github.com/p1g30n/explainshell-cli
or
> cargo uninstall explainshell-cli

Usually, it's a standard documentation process to tell users how to uninstall the package if they want. This is done by providing the necessary steps to uninstall the package. 

Unfortunately, we don't have any. If there is a way to clean and uninstall the package, then do let me know.

**Comment by thepigeonoftime** [comment]
Hm, that's unexpected behavior, `cargo uninstall explainshell-cli` should indeed work. 

I cannot reproduce the issue on my setup, does `cargo install --list` list the package for you? 
If not, `which explain` should reveal the path to the binary (on linux / macos). The default path should be `~/.cargo/bin/explain`.  Removing this file effectively uninstalls the program. 

Manual removal might leave traces in cargo's package registry (where it keeps track of installed packages), but if the package wasn't registered in the first place, I assume it's a non-issue. I'm not clear on cargo's internals in this regard, to be sure you could ask over at https://github.com/rust-lang/cargo .  

In general, is your rust toolchain on a recent version (`cargo --version` / `rustc --version`)?. The latest stable is 1.60.0 https://github.com/rust-lang/rust/releases

**Comment by PranavBhattarai** [comment]
> cargo uninstall explainshell-cli

This works pretty fine. I think I made a mistake because of the "guesswork" like this:
cargo --list  --------------------------------------------------------------------> _this doesn't actually show installed package name_
cargo uninstall explain -----------------------------------------------------> _this is not the original package name, it is command name_
cargo uninstall --git https://github.com/p1g30n/explainshell-c ---> _there is no such mechanism in existence_
```
  ⟹  cargo --version
cargo 1.60.0
  ⟹  rustc --version
rustc 1.60.0 (Fedora 1.60.0-1.fc36)
```
Regardless of my stupid mistake. We should have a simple uninstall command in **README** like we have for installation:

![image](https://user-images.githubusercontent.com/32763112/165004843-84210822-3c4e-425c-b73d-a3595e805329.png)

People can make stupid mistakes like me especially when they have to make a guess as I made.
